app/src/main/res/values-land/dimens.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
```

Comment:
As I said during out last call - just use default 'values' package for resources as currently we are not focusing on supporting different screen sizes of devices.